scripts/.env

```diff
@@ -10,6 +10,6 @@ MYSQL_USER=root
 MYSQL_PASSWORD=root
 MYSQL_DATABASE=itn
 
-START_BLOCK=0
+START_BLOCK=2355800
```

Comment:
What is reading this value? I wonder if this is interfering with the tests somehow.

Pushed a change to make this 0 again to see if that changes anything.